packages/@lwc/engine-core/src/framework/def.ts

```diff
@@ -130,10 +118,10 @@ function createComponentDef(
         errorCallback,
         render,
     } = proto;
-    const superProto = getCtorProto(Ctor, subclassComponentName);
+    const superProto = getCtorProto(Ctor);
     const superDef: ComponentDef =
         (superProto as any) !== BaseLightningElement
-            ? getComponentInternalDef(superProto, subclassComponentName)
+            ? getComponentInternalDef(superProto)
             : lightingElementDef;
     const SuperBridge = isNull(superDef) ? BaseBridgeElement : superDef.bridge;
```

Comment:
Based on the types, `superDef` can't be null here anymore.

```suggestion
    const SuperBridge = superDef.bridge;
```